.circleci/config.yml

```
@@ -320,7 +320,7 @@ jobs:
   www_deploy:
     <<: *defaults
     docker:
-      - image: circleci/ruby:2.6.0-node
+      - image: circleci/ruby:2.7.3
```

```suggestion
      - image: cimg/ruby:2.7.3
```

This is the newer set of images... note the name change from circleci to cimg https://circleci.com/developer/images/image/cimg/ruby